src/components/SimpleIconDropdown/SimpleIconDropdown.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown } from 'carbon-components-react';
+
+import { validThresholdIcons } from '../DashboardEditor/editorUtils';
+import { settings } from '../../constants/Settings';
+
+const { iotPrefix } = settings;
+
+const iconProptype = PropTypes.shape({
+  carbonIcon: PropTypes.any,
+  name: PropTypes.string,
+});
+
+const propTypes = {
+  /** Array of icons to be shown */
+  icons: PropTypes.arrayOf(iconProptype),
+  /** The title of the Dropdown, defaults to 'Icon' */
+  titleText: PropTypes.string,
+  /** Required Id string */
+  id: PropTypes.string.isRequired,
+  /** True if the light theme is to be used, defaults to false */
+  light: PropTypes.bool,
+  /** Callback for when any of the Dropdown icon value changes */
+  onChange: PropTypes.func.isRequired,
+  /** The selected icon, use to set initial icon */
+  selectedIcon: iconProptype,
+  /** Id used if needed for testing */
+  testID: PropTypes.string,
+};
+
+const defaultProps = {
+  icons: validThresholdIcons,
+  light: false,
+  selectedIcon: undefined,
+  testID: undefined,
+  titleText: '',
+};
+
+const SimpleIconDropdown = ({
```

Comment:
Is there a design spec for this component and has it been vetted accepted by the PAL design team? @dianatran18 
https://deploy-preview-1822--carbon-addons-iot-react.netlify.app/?path=/story/watson-iot-experimental-simpleicondropdown--default-example
![image](https://user-images.githubusercontent.com/3360588/99732210-b8d5c880-2a84-11eb-832e-afbd8d8bd416.png)

To my knowledge this functionality should fall into the [`IconDropdown`](https://deploy-preview-1822--carbon-addons-iot-react.netlify.app/?path=/story/watson-iot-experimental-icondropdown--default) pattern/component that's been verified by the PAL design team and is available. 

![image](https://user-images.githubusercontent.com/3360588/99732310-e589e000-2a84-11eb-8442-32d2b09e7a34.png)

@marcJV  If I recall correctly, it supports icons-only without the text labels. The color mod should also be supported through modifying the Icon passed to the dropdown?

Reply:
This was discussed with @CynZhang . Here is the design for the icon/color pickers as it stands for the dashboard editor implementation: 
![image](https://user-images.githubusercontent.com/53092833/100625321-fbe83500-32e9-11eb-91ae-b6a8cd2f4585.png)
![image](https://user-images.githubusercontent.com/53092833/100625274-ebd05580-32e9-11eb-8b0f-9492c2dfa09b.png)

We discussed using the 'simple/vertical' dropdowns in this PR for this early implementation, and then updating to these 'horizontal' ones in the next iteration.